src/client/app/utils/input.ts

```diff
@@ -82,6 +82,8 @@ export const NoUnit: UnitData = {
 	suffix: '',
 	displayable: DisplayableType.none,
 	preferredDisplay: false,
+	defaultMeterMinimumValue: 0,
+	defaultMeterMaximumValue: 0,
```

Comment:
For no unit, the values should be -Infinity,Infinity. Please refer to the meeting recording for any clarification